sql-plugin/src/main/330+/scala/org/apache/spark/sql/catalyst/json/rapids/shims/Spark33XFileOptionsShims.scala

```diff
@@ -16,12 +16,12 @@
 
 package org.apache.spark.sql.catalyst.json.rapids.shims
 
-import com.nvidia.spark.rapids.shims.Spark321PlusShims
+import com.nvidia.spark.rapids.shims.{Spark320PlusNonDBShims, Spark321PlusShims}
 
 import org.apache.spark.sql.catalyst.csv.CSVOptions
 import org.apache.spark.sql.catalyst.json.JSONOptions
 
-trait Spark33XFileOptionsShims extends Spark321PlusShims {
+trait Spark33XFileOptionsShims extends Spark321PlusShims with Spark320PlusNonDBShims {
```

Comment:
nit: I personally think this would be better as an object in the v2 shims instead of a trait here. Before the names of the traits matched with what was expected, but now we have a new file for databricks 2.1 where the code matches this, but is named like the other ones.